### ISSUE TITLE
Add Visualizations and RealTime updates

### ISSUE THREAD

**Issue by MrClan** [MrClan/RealTimePolls#3]
An intuitive visualization which is updated in real time as and when the votes are casted.

**Comment by bverpaalen** [comment]
I think [this](https://bl.ocks.org/mbostock/3885304) kind of visualizations would be the best.
The y-as could show the amount of votes
The x-as could show the options
Maybe have an extra number what show the total votes?

**Comment by MrClan** [comment]
Or maybe we can display it in a chart instead of xy graph, since options for values may not be very intuitive. Think about cases, where options may include people-photos. A chart with scattered images, and with no. of votes would look good and less technical.

**Comment by bverpaalen** [comment]
Something like [this](https://bl.ocks.org/mbostock/3883245) for every option?
Or a line for every option and have legenda with the options?

**Comment by MrClan** [comment]
Not exactly, but something like this one: http://i.imgur.com/g0kNXFB.png
The image maybe scattered instead of aligned in a grid.

**Comment by bverpaalen** [comment]
So in this case Shri Nandlal would have the highest votes and Dr. Vassant Ramji would have the lowest?

**Comment by MrClan** [comment]
yes. But that's just a reference image. I'm unable to find a suitable image. Please see this one:
 https://lh3.googleusercontent.com/OHBbYFuAq1hBt3wdyLyGVCxsv1lAXipEybWb45nPBPtR-GanY5oUMj8Qgs-he9QxBoBx=h900

Here the background would be a full page browser window. And instead of app icons, there can be the poll-entities(or options) with numbers indicating vote count(like facebook messages count) :)

**Comment by MrClan** [comment]
looks complicated, but visually anything close to this would be awesome: http://www.cotrino.com/lifespan/

PS: 
http://vax.herokuapp.com/tour [yet another relevant an cool visualization]
http://bl.ocks.org/phuonghuynh/54a2f97950feadb45b07
